internal/util/util.go

```diff
@@ -153,9 +157,12 @@ func GetResourceOwner(owner *string) (string, string, error) {
 	if owner == nil {
 		return "", "", fmt.Errorf("owner string is nil")
 	}
+	if strings.HasPrefix(*owner, "error:") {
+		return "error", *owner, nil
+	}
```

Comment:
can we document why this is the case?

Reply:
This code is outdated, from a previous version where I set the owner to "error:multiple" if the device had multiple owners.  Removed.